FAQ Update - How can I optimise my project to speed it up? (#212)
subquery/documentation: docs/faqs/faqs.md

```

Note that it is recommended to use `--force-clean` when changing the `startBlock` within the project manifest (`project.yaml`) in order to begin reindexing from the configured block. If `startBlock` is changed without a `--force-clean` of the project, then the indexer will continue indexing with the previously configured `startBlock`.

## How can I optimise my project to speed it up?

Performance is a crucial factor in each project. Fortunately, there are several things you could do to improve it. Here is the list of some suggestions:

- Avoid using block handlers where possible.  
- Query only necessary fields. 
- Try to use filter conditions to reduce the response size. Create filters as specific as possible to avoid querying unnecessary data. 
- For large data tables, avoid querying `totalCount` without adding conditions.

> - Always add index to schema for the historical project.
> - Add indexes to entity fields for query performance. 

Reviewer: ```suggestion
- Add indexes to entity fields for query performance, this is especially important for historical projects
```
